### PR TITLE
Add a bio file for @github_vijaya

### PR DESCRIPTION
**Pull request by vijaya14**
This pull request adds a bio file for @github_vijaya-bio as requested in #_vijaya-bio

@githubteacher

### CODE REVIEW

File: docs/githubvijaya.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# GitHub Vijaya Bio
+
+- **Name:** GitGub Vijaya
+- **Occupation:** Vijaya
+- **Location:** San Diego
```
Comment (vijaya14) [inline docs/githubvijaya.md]: please specify state.

**Comment by IPandaType** [comment]
Hey Vijaya! Nice work!

File: docs/githubvijaya.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# GitHub Vijaya Bio
+
+- **Name:** GitGub Vijaya
+- **Occupation:** Vijaya
```
Comment (firdausirani) [inline docs/githubvijaya.md:4]: What is the baseline salary for this Occupation?

File: docs/githubvijaya.md (hunk 1 above)
Comment (IPandaType) [inline docs/githubvijaya.md]: Nice work!

**Comment by vijaya14** [comment]
go for it.

**Pull request merged**